### ISSUE TITLE
some of foreignNames contain extra ' —' in AKH

### ISSUE THREAD

**Issue by Agjayd** [mtgjson/mtgjson#354]
Ancient Crab(AKH)
… "foreignNames": [
        {
          "language": "Chinese Simplified",
          "name": "远古蟹 —",
          "multiverseid": 429163
        },…

seems to happen to all Chinese Japanese and French names of creatures in AKH

**Comment by naokiy** [comment]
I think this is a problem of http://magiccards.info/.
( see http://magiccards.info/akh/jp/1.html )

I sent a request to fix it via the contact form.

**Comment by Garbee** [comment]
The contact form will probably end up in an issue here anyways. If you know to use Github issues it is better than the contact form at getting things looked at.

**Comment by naokiy** [comment]
Now, the problem seems to be fixed at http://magiccards.info/.
So I think if we build JSON again, we will get correct card name.

**Comment by bra1n** [comment]
Any updates on this? Gatherer also does not have the erroneous extra dash in the foreign names:
http://gatherer.wizards.com/Pages/Card/Languages.aspx?multiverseid=427657
```
{
"language": "French",
"name": "Mage scarifiâme —",
"multiverseid": 427657
}
```

**Comment by naokiy** [comment]
I recreate json and create MR at #372 .
When it's merged and released, this problem will be fixed.

**Comment by naokiy** [comment]
This problem seemed to be solved in v3.8.6 .
